bundles/org.openhab.binding.daikinairbase/src/main/java/org/openhab/binding/daikinairbase/internal/handler/DaikinAirbaseAcUnitHandler.java

```diff
@@ -0,0 +1,349 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.daikinairbase.internal.handler;
+
+import java.io.IOException;
+import java.util.Optional;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import javax.measure.quantity.Temperature;
+
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.library.types.QuantityType;
+import org.eclipse.smarthome.core.library.types.StringType;
+import org.eclipse.smarthome.core.library.unit.SIUnits;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.types.RefreshType;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.State;
+import org.eclipse.smarthome.core.types.UnDefType;
+import org.openhab.binding.daikinairbase.internal.DaikinAirbaseBindingConstants;
+import org.openhab.binding.daikinairbase.internal.DaikinAirbaseCommunicationException;
+import org.openhab.binding.daikinairbase.internal.DaikinAirbaseWebTargets;
+import org.openhab.binding.daikinairbase.internal.api.ControlInfo;
+import org.openhab.binding.daikinairbase.internal.api.Enums.FanMovement;
+import org.openhab.binding.daikinairbase.internal.api.Enums.FanSpeed;
+import org.openhab.binding.daikinairbase.internal.api.Enums.Mode;
+import org.openhab.binding.daikinairbase.internal.api.SensorInfo;
+import org.openhab.binding.daikinairbase.internal.api.ModelInfo;
+import org.openhab.binding.daikinairbase.internal.api.ZoneInfo;
+import org.openhab.binding.daikinairbase.internal.config.DaikinAirbaseConfiguration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Handles communicating with a Daikin Airbase air conditioning unit.
+ *
+ * @author Tim Waterhouse - Initial Contribution
+ *
+ */
+public class DaikinAirbaseAcUnitHandler extends BaseThingHandler {
+
+    private final Logger logger = LoggerFactory.getLogger(DaikinAirbaseAcUnitHandler.class);
+
+    private long refreshInterval;
+
+    private DaikinAirbaseWebTargets webTargets;
+    private ScheduledFuture<?> pollFuture;
+
+    public DaikinAirbaseAcUnitHandler(Thing thing) {
+        super(thing);
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
```

Comment:
You should also handle REFRESH commands within this method.

Reply:
aebd553945 should add support for REFRESH